wdl2cwl/main.py

```diff
@@ -433,28 +433,37 @@ def get_outdir_requirement(
                 isinstance(outdir, (WDL.Expr.Apply, WDL.Expr.String))
                 and outdir.literal is None
             ):
+                # runtime["disks"] usually have strings chars like local, HDD, SDD that are not needed by CWL
                 if (
                     isinstance(outdir, WDL.Expr.Apply)
                     and not outdir.function_name == "_add"
                 ):
+                    # If it contains an apply expr we don't want to process the _add function
+                    # that concatenates it to the chars in the string
                     expr_str = self.get_expr(outdir)
+                    # if the expr_str returns an integer that is declared as a static variable
+                    # return the result from the multiplication with 1024
```

Comment:
We can see that the result is multiplied by 1024, so don't repeat yourself; better to say why we multiply by 1024